### PR TITLE
env: update eslint environment with vscode editor

### PR DESCRIPTION
**Pull request by JaeYeopHan**

### CODE REVIEW

File: .config/rome.rjson [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+lint: {
+	ignore: [
+		".yarn",
+		"pnp.js",
+		"packages/**/dist/*",
+		"packages/**/lib/*",
+		"packages/**/esm/*"
+	]
+}
+root: true
```
Comment (SoYoung210) [inline .config/rome.rjson:12]: eof~

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[*]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+charset = utf-8
+indent_style = space
```
Comment (SoYoung210) [inline .editorconfig:6]: oh,,,,,,no.,,! why not just eslint?

Comment (JaeYeopHan) [reply .editorconfig:6]: for rome (in future)

Comment (SoYoung210) [reply .editorconfig:6]: is rome use editorconfig only?

Comment (JaeYeopHan) [reply .editorconfig:6]: yes

File: .config/rome.rjson (hunk 1 above)
Comment (JaeYeopHan) [reply .config/rome.rjson]: resolved

**Pull request merged**